MakingItBetterG55/src/application/Player.java

```diff
@@ -0,0 +1,64 @@
+package application;
+
+public class Player {
+	
+	private char playerType;
+	private String playerName;
+	private boolean aiControlled;
+	private int money;
+	private int health;
+	private int education;
+	private int goods;
+	
+	public Player(char playerType, String playerName, boolean aiControlled, 
```

Comment:
since the resource amounts depend on the type, remove the resources as arguments and have them set in a switch based on type

Reply:
Done.